getting-started/plugin-features/README.md

```diff
@@ -3,101 +3,96 @@
 The following features are supported by the plugin, as listed below.
 
 - [Running Ballerina Programs](#running-ballerina-programs)
-- [Debugging Ballerina Programs](#debugging-ballerina-programs) 
-- [Diagram View](#diagram-view)
 - [Import modules On-the-fly](#import-modules-on-the-fly)
-- [Auto Import Unambiguous Modules](#auto-import-unambiguous-modules)
 - [Find Usage](#find-usage)
 - [Go to Definition](#go-to-definition)
 - [Formatting](#formatting)
-- [Parameter Information](#parameter-information)
+- [Signature help](#signature-help)
 - [Quick Documentation](#quick-documentation)
-- [Object/Record Fields suggestions](#objectrecord-fields-suggestions)
-- [Annotation fields suggestions](#annotation-fields-suggestions)
+- [Code completions](#code-completions)
+- [Code snippets](#code-snippets)
+- [Code actions](#code-actions)
+- [Error diagnostics](#error-diagnostics)
 - [File Templates](#file-templates)
-- [Live Templates and Code snippets](#live-templates-and-code-snippets)
 - [Spell Checking](#spell-checking)
-- [Semantic Analyzing](#semantic-analyzing) 
 
-## Running Ballerina Programs 
+## Running Ballerina Programs
 
 You can run Ballerina main/service programs with only few clicks. You don't have to add or change any configurations.
+You can click on the gutter icon, right click menu or the run button
+to execute a ballerina program.
 
-![alt text](images/RunConfig.gif)
+![alt text](images/run.gif)
 
-## Debugging Ballerina Programs 
+You can also run ballerina tests easily like running ballerina programs with right click actions and gutter icons.
```

Comment:
Shall we mention this running ballerina test support under a H3 header instead? also you can mention that we support running individual tests as well as, running all the module tests at once.